src/tfvc/commands/commandhelper.ts

```diff
@@ -89,6 +89,9 @@ export class CommandHelper {
             }
             lines = lines.splice(index);
         }
+        if (filterEmptyLines) {
+            lines = lines.filter(e => e !== "");
```

Comment:
you may want to trim() here to remove "blank" lines as well. Not sure if you want to or not.